AppSettingsManagerAPI/AppSettingsManagerApi.Domain/MySql/IBaseUserRepository.cs

```diff
@@ -1,11 +0,0 @@
-using AppSettingsManagerApi.Model;
-
-namespace AppSettingsManagerApi.Domain.MySql;
-
-public interface IBaseUserRepository
-{
-    Task<BaseUser> GetUser(string userId);
-    Task<BaseUser> CreateUser(string userId, string password, string email);
-    Task<BaseUser> DeleteUser(string userId);
-    Task<BaseUser> UpdateUser(string userId, string newPassword);
-}
```

Comment:
are these new CRUD commands??

Reply:
They're only showing as being deleted because the lines changed. All that changed is the name of the class in the return type